### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by roniemartinez**
| Q                | A
| ---------------- | ---
| Bug fix?         | yes
| New feature?     | no
| BC breaks?       | no
| Deprecations?    | no
| New tests added? | not needed
| Fixed tickets    | #1503 
| License          | MIT

### Description

- Update packages
- Fix tests
- Fix cssmin warnings
- Change jscs to eslinit
- Change csslint to stylelint
- Fix spellings
- Swap css and jss tasks

### CODE REVIEW

**Comment by sauravkhdoolia** [comment]
@roniemartinez What about this one?
Merge it?

**Comment by roniemartinez** [comment]
@sauravkhdoolia Yeah I think I almost forgot this since no one reviewed it before. Will merge it.

File: Gruntfile.js [hunk 1]
```diff
@@ -172,20 +169,10 @@ module.exports = function (grunt) {
         }
     };
 
-    gruntConfig.csslint = {
```
Comment (sauravkhdoolia) [inline Gruntfile.js:175]: Why are we removing these linter configs?
I think thay were probably added for consistency in CSS too!

File: dist/js/medium-editor.js [hunk 2]
```diff
@@ -182,7 +182,7 @@ if (!("classList" in document.createElement("_"))) {
  */
 
 /*global self, unescape */
-/*jslint bitwise: true, regexp: true, confusion: true, es5: true, vars: true, white: true,
+/*jslint bitwise: true, regexp: true, confusion: true, vars: true, white: true,
```
Comment (sauravkhdoolia) [inline dist/js/medium-editor.js:185]: jslint??

File: Gruntfile.js (hunk 1 above)
Comment (roniemartinez) [reply Gruntfile.js:175]: CSSLint is not maintained anymore
Here is the related changes: https://github.com/yabwe/medium-editor/issues/1503#issuecomment-552683079

File: dist/js/medium-editor.js (hunk 2 above)
Comment (roniemartinez) [reply dist/js/medium-editor.js:185]: ESLint

**Pull request closed without merging**